grpc/src/main/java/com/linecorp/armeria/server/grpc/ArmeriaServerCall.java

```diff
@@ -193,6 +194,9 @@
         this.compressorRegistry = requireNonNull(compressorRegistry, "compressorRegistry");
         clientAcceptEncoding =
                 Strings.emptyToNull(clientHeaders.get(GrpcHeaderNames.GRPC_ACCEPT_ENCODING));
+        if (autoCompression && clientAcceptEncoding != null) {
+            setCompression(clientAcceptEncoding);
```

Comment:
`clientAcceptEncoding` is comma separated values.
> Message-Accept-Encoding → "grpc-accept-encoding" Content-Coding *("," Content-Coding)
https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#requests

Could we take the first encoding that `compressorRegistry` has as the default?

Reply:
Thanks for your review.
I addressed it.